resources/views/emails/dailyhelpermail.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('emails.template')
+
+@section('body')
+
+    <p>
+        Hey {{ $user->calling_name }},
+    </p>
+
+    <p>
+        You receive this e-mail because you are needed by one or more activities! The activities that need your help are
+        listed below.
+    </p>
+
+    @php $i = 0; @endphp
+
+    @foreach($events as $eventId => $event)
+
+        @php
+
+            // This is done in this template in order to prevent the jobs table from containing large amounts of event data.
+            $event->event = Proto\Models\Event::find($eventId);
```

Comment:
Please include withTrashed so the mail does not fail if an event is deleted after dispatching the mail.

Reply:
Also, Event is aliased so you don't need to include the whole namespace.